src/tests/Missions.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import Missions from '../components/Missions';
+
+import store from '../Redux/ConfigureStore';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Provider store={store}><Missions /></Provider>).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('Render Missions component', () => {
+  render(<Provider store={store}><Missions /></Provider>);
+  screen.debug();
+});
```

Comment:
- [ ] [Optional] Your test is failing, I am making this optional because the previous reviewer did not mention it, but I strongly recommend you fix this
![Screenshot from 2022-07-21 12-47-51](https://user-images.githubusercontent.com/66097371/180206957-1a44b229-616a-4c76-bc19-dcffabe95bc0.png)